grazer/src/main/webapp/WEB-INF/spring/root-context.xml

```diff
@@ -26,7 +40,12 @@
 		<property name="basename" value="classpath:locale/messages" />
 		<property name="fallbackToSystemLocale" value="false" />
 	</bean>
-
+	
+	<bean id="transactionManager"
+        class="org.springframework.orm.hibernate4.HibernateTransactionManager">
+        <property name="sessionFactory" ref="sessionFactory" />
+    </bean>
```

Comment:
why are there two HibernateTransactionManger (one here, one in spring-dbconnectionmanager.xml)?